assets/quantity-popover.css

```diff
@@ -0,0 +1,78 @@
+quantity-popover volume-pricing li:nth-child(odd) {
+  background: rgba(var(--color-foreground), 0.03);
+}
+
+quantity-popover volume-pricing li {
+  font-size: 1.4rem;
+  letter-spacing: 0.06rem;
+  padding: 0.8rem 1.2rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.quantity-popover__info.global-settings-popup {
+  position: absolute;
+  width: 100%;
+  z-index: 2;
+  background-color: rgb(var(--color-background));
+}
+
+.quantity-popover__info--wide {
+  padding-top: 3.2rem;
+}
+
+.quantity-popover__info .button {
+  width: 4.4rem;
+  height: 4.4rem;
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 0 0.8rem 0 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.quantity-popover__info .button .icon {
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+quantity-popover volume-pricing {
+  margin-top: 1.2rem;
+  display: block;
+}
+
+quantity-popover .quantity__rules span:first-of-type {
```

Comment:
Is there a way to target the same element without making assumptions about where it is in the DOM?

Reply:
So we dont know which one will be the first one. It could be that they have min, max and increment set up or just min or max. Or just min. But the first element will need to be full width as per the designs

Let me know if this makes sense